backend/src/main/java/com/eyetrade/backend/model/entity/Event.java

```diff
@@ -0,0 +1,48 @@
+package com.eyetrade.backend.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by Emir Gökdemir
+ * on 7 Kas 2019
+ */
+@Entity
+@Data
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "event")
+public class Event {
```

Comment:
That table has duplicate data with the "rss_feed_event" table. Therefor, I think maybe we should discuss about that design later to make it more storage efficient.

Reply:
Some rss_feed_event cannot take event, so I store them in order to know whether update is occured or not

Reply:
I understand now. Good job.